netci.groovy

```diff
@@ -1610,6 +1627,10 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                         armCrossgenOpt = '-altjitcrossgen'
                     }
 
+                    if (scenario == 'pri1') {
```

Comment:
This doesn't seem necessary, since at the top there is:
```
    if (scenario == 'default' && isPR == true) {
        priority = '0'
    }
```
so priority will already be 1 for scenario==pri1.